_rules/SC2-2-1+SC3-2-5-meta-refresh.md

```diff
@@ -1,77 +1,157 @@
-
 ## Test procedure
 
-### Selector
-
-Select all elements that match the following CSS selector:
-
-    meta[http-equiv="refresh"][content]
-
-For each selected item, go through the following steps:
-
-### Step 1
-
-Take the value of the content attribute of the selected element.
-
-Remove any characters starting after the first comma or semicolon from the value.
-
-Parse the remainder to an integer.
-
-If the integer is invalid or 0, return [step1-pass](#step1-pass)
-
-Else return [step1-fail](#step1-fail)
-
-## Outcome
-
-The resulting assertion is as follows,
-
-| Property | Value
-|----------|----------
-| type     | Assertion
-| test     | auto-wcag:{{ page.rule_id }}
-| subject  | *the selected element*
-| mode     | auto-wcag:{{ page.test_mode }}
-| result   | <One TestResult from below>
-
-### step1-pass
-
-| Property    | Value
-|-------------|----------
-| type        | TestResult
-| outcome     | Passed
-| description |
-
-### step1-fail
-
-| Property    | Value
-|-------------|----------
-| type        | TestResult
-| outcome     | Failed
-| description | Meta refresh should not be used unless it is instantaneous.
+### Applicability
+
+The rule applies to the first [valid](https://www.w3.org/TR/html/document-metadata.html#statedef-http-equiv-refresh) `<meta http-equiv="refresh">` element with a `content` attribute in a document.
+
+### Expectation
+
+The `time` of the `content` attribute is not between 0 and 72000 (20 hours).
+
+**Note**: See [Refresh state (`http-equiv="refresh"`)](https://www.w3.org/TR/html/document-metadata.html#statedef-http-equiv-refresh) for a precise description on how to determine the `time`.
+
+## Assumptions  
+
+* This test assumes no functionality was provided by the website for the user to adjust the timer. 
+* This test assumes that the refresh was not [essential](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-required-behaviors.html#essentialdef), which is listed as a valid exception to SC 2.2.1.
+
+## Accessibility Support 
+
+There are no major accessibility support issues known for this rule.
+
+## Background  
+
+- H76: Using meta refresh to create an instant client-side redirect
+- F40: Failure of Success Criterion 2.2.1 and 2.2.4 due to using meta redirect with a time limit
+- F41: Failure of Success Criterion 2.2.1, 2.2.4, and 3.2.5 due to using meta refresh with a time-out
+
+## Test Cases
+
+### Passed
+
+```html
+<!-- redirects immediately -->
+  <head>           
+    <meta http-equiv="refresh" content="0; URL='https://auto-wcag.github.io/auto-wcag/'" />    
+  </head>  
+```
+
+```html
+<!-- first valid <meta http-equiv="refresh"> redirects immediately  -->
+<head>
+  <meta http-equiv="refresh" content="0; http://example.com" />
+  <meta http-equiv="refresh" content="5; http://example.com" />
+</head>
+```
+
+```html
+<!-- redirects immediately -->
+<head>
+  <meta http-equiv="refresh" content="0 http://example.com" />
+</head>
+```
+
```

Comment:
I think the test case above is a duplicate from the first test case. Apart from this minor thing I approve the rule.

Reply:
It's not - it's a broken test case. Removing it. Good catch.